src/components/SubTypeAccordion/SubTypeAccordionsContainer.js

```diff
@@ -36,16 +33,11 @@ const SubTypeAccordionsContainer = ({ subTypeData, fromContentPanel }) => (
 );
 
 SubTypeAccordionsContainer.propTypes = {
-  subTypeData: shape({}).isRequired,
-  fromContentPanel: bool,
-};
-
-SubTypeAccordionsContainer.defaultProps = {
-  fromContentPanel: false,
+  accordionsContainerData: shape({}).isRequired,
 };
 
-const mapStateToProps = (state) => ({
-  resourceTypeFilters: state.resourceTypeFilters,
+const mapStateToProps = (state, ownProps) => ({
+  accordionsContainerData: accordionsContainerDataSelector(state, ownProps),
```

Comment:
it seems `fromContentPanel` and `fromCatalogScreen` escape propType validation, when used via `mapStateToProps`

I wonder if there's an eslint rule that can detect this via the caller/parent component that passes the props.

Reply:
couldn't find a rule upon quick google search. perhaps this is a refactor later. Or I can add the props validation and add `react/no-unused-prop-types` with a note